tests/adapters/switches/dell_test.py

```diff
@@ -1163,69 +1033,7 @@ def test_configure_bond_native_vlan_on_a_trunk_mode_swithes_to_general_and_copie
 
         self.switch.configure_bond_native_vlan(10, 1000)
 
-    def test_remove_bond_native_vlan_reverts_to_trunk_mode(self):
-        self.command_setup()
-
-        self.mocked_ssh_client.should_receive("do").with_args("show running-config interface port-channel 10").and_return([
-            "switchport mode general",
-            "switchport general pvid 1000"
-        ])
-
-        with self.configuring_and_committing():
-            self.mocked_ssh_client.should_receive("do").with_args("interface port-channel 10").once().ordered().and_return([])
-            self.mocked_ssh_client.should_receive("do").with_args("switchport mode trunk").once().ordered().and_return([])
-            self.mocked_ssh_client.should_receive("do").with_args("exit").once().ordered().and_return([])
-
-        self.switch.remove_bond_native_vlan(10)
-
-    def test_remove_bond_native_vlan_reverts_to_trunk_mode_and_keeps_allowed_vlans_specs(self):
-        self.command_setup()
-
-        self.mocked_ssh_client.should_receive("do").with_args("show running-config interface port-channel 10").and_return([
-            "switchport mode general",
-            "switchport general pvid 1000",
-            "switchport general allowed vlan add 1201-1203,1205",
-            ])
-
-        with self.configuring_and_committing():
-            self.mocked_ssh_client.should_receive("do").with_args("interface port-channel 10").once().ordered().and_return([])
-            self.mocked_ssh_client.should_receive("do").with_args("switchport mode trunk").once().ordered().and_return([])
-            self.mocked_ssh_client.should_receive("do").with_args("switchport trunk allowed vlan add 1201-1203,1205").once().ordered().and_return([])
-            self.mocked_ssh_client.should_receive("do").with_args("exit").once().ordered().and_return([])
-
-        self.switch.remove_bond_native_vlan(10)
-
-    def test_remove_bond_native_vlan_inknown_interface(self):
```

Comment:
This looks like a previous test_remove_bond_native_vlan_inknown_interface, but this one asserted an UnknownBond. I think it should simply be renamed.